Projects docs (#237)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/index.mdx

## Listing and Filtering Workspaces

Click **Projects & workspaces**. Terraform Cloud displays the workspaces the current user account has permission to read runs for. The project drawer lists any projects the user has access to. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))

[permissions-citation]: #intentionally-unused---keep-for-maintainers

If your organization contains many workspaces, you can use the filter tools at the top of the list to find the workspaces you are interested in.

The following filters are available:

- **Project filter:** Click the left chevron icon next to **Workspaces** to open the project drawer. Clicking on a project will filter the workspaces list to show only the workspaces in the selected project. If your organization has many projects, click the search icon next to **Projects** to find a project.

> - **Run Status filter buttons:** These filters show workspaces by the status of their current run. There are five quick filter buttons that collect the most commonly used groups of statuses: Needs Attention, Errored, Running, On Hold, and Applied).

Reviewer: ```suggestion
- **Run Status filter buttons:** These filters display workspaces with the selected run status. There are five quick filter buttons that collect the most commonly used groups of statuses: Needs Attention, Errored, Running, On Hold, and Applied.
```
